Tests/OpenCombineTests/CurrentValueSubjectTests.swift

```diff
@@ -441,13 +478,16 @@ final class CurrentValueSubjectTests: XCTestCase {
 
         race(
             {
+                lock.lock()
                 cvs.value += 1
+                lock.unlock()
             },
             {
+                lock.lock()
                 cvs.value -= 1
+                lock.unlock()
```

Comment:
Maybe we should replace `cvs.value += 1` with just `cvs.send(42)`, and also remove this locking? It feels like it makes this test useless, since the access is guaranteed to be synchronized at test site, and we want to test the synchronization of `CurrentValueSubject` itself.

Reply:
I removed the lock and just send 42 now. The thread sanitiser should catch if we have a sync problem here I guess.

Reply:
Thanks! Does the sanitizer complain if you remove the lock from the `value` getter and setter?